boards/teensy31/Makefile.features

```diff
@@ -1,11 +1,13 @@
 # Put defined MCU peripherals here (in alphabetical order)
-FEATURES_PROVIDED += periph_cpuid
-FEATURES_PROVIDED += periph_gpio
 FEATURES_PROVIDED += periph_pwm
+FEATURES_PROVIDED += periph_rtc
+FEATURES_PROVIDED += periph_rtt
```

Comment:
Is this related ?

Reply:
I had to pull in some upstream merged PRs to avoid problems rebasing later.